Rakefile

```diff
@@ -1,3 +1,10 @@
+def is_in_a_docker?
+  `grep -q docker /proc/self/cgroup`
+  $?.success?
+end
+
+abort("Not in a Docker, you should type \"docker-compose run <task>\".") unless is_in_a_docker?
```

Comment:
*nitpick*: "Not running in docker, ..." lowercase because we're talking about the container runtime and not the company.